scripts/tests/aux-functions.sh

```diff
@@ -15,5 +15,6 @@ test_package_with_feature() {
     local features=$2
 
     /bin/echo -e "\e[0;33m***** Testing '$package' with features '$features' *****\e[0m\n"
-    cargo test --features $features --manifest-path $package/Cargo.toml --no-default-features --release
```

Comment:
We should avoid profile release for tests, so that `debug_assert!` macros are activated and we can see, that something is not as expected.